driver-core/src/main/com/mongodb/connection/ClusterSettings.java

```diff
@@ -268,6 +268,19 @@ public Builder addClusterListener(final ClusterListener clusterListener) {
             return this;
         }
 
+        /**
+         * Sets the cluster listeners.
+         *
+         * @param clusterListeners list of cluster listeners
+         * @return this
+         * @since 4.5
+         */
+        public Builder clusterListenerList(final List<ClusterListener> clusterListeners) {
```

Comment:
@stIncMale I'm wondering if, for consistency with the getter, this method should be named `clusterListeners` instead of `clusterListenerList`.  The "List" is implied by the type.  What do you think?

Reply:
> The "List" is implied by the type

I agree, but the PR description states that this method was modelled after [`MongoClientSettings.Builder.commandListenerList`](https://mongodb.github.io/mongo-java-driver/4.4/apidocs/mongodb-driver-core/com/mongodb/MongoClientSettings.Builder.html#commandListenerList(java.util.List)) (there is also [MongoClientSettings.Builder.compressorList](https://mongodb.github.io/mongo-java-driver/4.4/apidocs/mongodb-driver-core/com/mongodb/MongoClientSettings.Builder.html#compressorList(java.util.List))). If you are OK with breaking this consistency, then I have no objections to naming new `*List` methods differently.

Reply:
Ah, I see.  Let's leave as is then.